src/components/item-card/ItemCard.tsx

```diff
@@ -13,12 +13,21 @@ type Props = {
   isFavorite: boolean
 }
 
+const MAX_DESCRIPTION_LENGTH = 69
+
 const ItemCard: FC<Props> = ({ item, isFavorite }) => {
   const [selectedWeight, setSelectedWeight] = useState(item.prices[0].weight)
   const [selectedPrice, setSelectedPrice] = useState(item.prices[0].price)
 
   const dispatch = useDispatch()
 
+  const truncatedDescription =
+    item.description.length > MAX_DESCRIPTION_LENGTH
+      ? `${item.description.slice(0, MAX_DESCRIPTION_LENGTH)}...`
+      : item.description
+
+  const checedWeight = (weight: string) => (weight === '1000' ? '1кг' : weight)
```

Comment:
what is `chaced`? maybe the better name for function would be `formatWeight`? And also you should move it to `utils/format-weight.ts` and make it more usable for other weight

like this one (you can change it, that is just sketch)

```
const gramsToKilograms = (grams) => {
  const kilograms = Math.round(grams / 1000);
  return kilograms;
}
```